src/components/VButton.vue

```diff
@@ -10,7 +10,7 @@
       isActive && $style[`${variant}-pressed`],
       $style[`size-${size}`],
       isPlainDangerous
-        ? ''
+        ? 'border border-tx'
         : 'border border-tx focus-visible:ring focus-visible:ring-pink',
```

Comment:
'plain--avoid' is also used in these places. Changing the base style could have unforeseen changes that are not captured in e2e.

- `src/components/VHeader/VHeaderMobile/VSearchBarButton.vue`
- `src/components/VTabs/VTab.vue`